File: CS/WindowsFormsApplication1/Form_XtraTabbedMDIManager.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace WindowsFormsApplication1 {
    public partial class Form_XtraTabbedMDIManager : DevExpress.XtraBars.Ribbon.RibbonForm {
        public Form_XtraTabbedMDIManager() {
            InitializeComponent();
        }

        private void barButtonItem1_ItemClick(object sender, DevExpress.XtraBars.ItemClickEventArgs e) {
            ReportPreviewHelper.ShowReport(this);
        }
    }
}

File: VB/WindowsFormsApplication1/Form_XtraTabbedMDIManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq

Namespace WindowsFormsApplication1
    Partial Public Class Form_XtraTabbedMDIManager
        Inherits DevExpress.XtraBars.Ribbon.RibbonForm

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub barButtonItem1_ItemClick(ByVal sender As Object, ByVal e As DevExpress.XtraBars.ItemClickEventArgs) Handles barButtonItem1.ItemClick
            ReportPreviewHelper.ShowReport(Me)
        End Sub
    End Class
End Namespace
